server/app_multi/app.py

```diff
@@ -0,0 +1,163 @@
+import os
```

Comment:
Why do we need both app_single and app_multi?   Naively, I would have thought the only difference was that the "app single" case needs a default dataset in its routes (in other words the '/' route will serve the default dataset), whereas the "app multi" case has no default dataset.

I don't see a world where we need to _restrict_ the app to serving only a single dataset (except maybe for infosec reasons -- which we should think through).

Reply:
I merged all the app_single and app_multi code into one thing, and removed the distinction between launch and launch-multi and the configuration for single and multi.
You can now launch with a datafile and/or a dataroot.